src/components/Text/Text.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+import { fontSmall } from '../../styles/designTokens';
+
+const Text = styled.span`
+  font-size: ${fontSmall};
+  text-decoration: ${({ isUnderline }) => (isUnderline ? 'underline' : 'none')};
```

Comment:
As per previous comments change to **underline** and **bold** (I know it is good idea to prefix flags with 'is' but in styled-components it is common pattern to not do it).